### DISCUSSION TITLE
2.3.60 docker containers missing

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#4762] [Unsupported Versions]
Doing a distributed (true clustering) airgap upgrade, after a while soup stated the salt-master could not be reached. I believe I then ran soup again and got further, but so-elastalert and so-elasticsearch containers are now missing. Docker images does not show these two 2.3.60 containers. I am looking for any advice on troubleshooting.

Very excited for the elastic authentication integration!

**Comment by darth-drew** [comment]
Update: I have found in /nsm/docker-registry that soup was able to create some so-elasticsearch 2.3.60 files, but am so far unsuccessful in finding where or if the files required to build the container were created. 

In parallel to this, I am running the 2.3.60 upgrade on a manager that I had in "standby" that is not connected to the network at this time, and it appears the update process was hung at the starting salt-master service again, however so-status is reporting all green. I ran a "docker image save" on the two containers that were reporting issues on the primary manager, and copied those containers to the primary.

Does anyone have recommendations on how to spin up these containers, so that tools like so-status will show everything is working properly?

**Reply by bicfordio** [reply]
I don't know if you ended up solving your issues, but for me it was due to random issues that the soup script applied. https://github.com/Security-Onion-Solutions/securityonion/discussions/4859
